### PR TITLE
Extract pylint workflow

### PR DESCRIPTION
**Pull request by Fahi28**
Pylint workflow is working. Added better type hint aswell.

### CODE REVIEW

File: text_extract.py
Comment (bentrzcinski) [inline text_extract.py:1]: this should be called 'test_extract.py'

**Review by bentrzcinski — COMMENTED** [review]
otherwise all dandy

File: text_extract.py
Comment (Fahi28) [reply text_extract.py:1]: thanks for that

**Review by anitamegarry — APPROVED** [review]
Looks great, thanks!

**Review by jiuliangut — APPROVED** [review]
nice :)

**Pull request merged**